conftest.py

```diff
@@ -13,6 +13,22 @@
 # limitations under the License.
 
 
+def pytest_configure(config):
+    # Ignore deprecation warnings in python code generated from our protobuf definitions.
+    # Eventually, the warnings will be removed by upgrading protoc compiler. However, at
+    # the moment upgrading breaks TensorFlow Quantum and therefore is deferred. See issues
+    # #4161 and #4737.
```

Comment:
TensorFlow Quantum is no longer blocking here. It is the fact that nobody has bothered to do a GAPIC rebuild of the v1alpha1 protos.

Reply:
Updated comment. PTAL